react-app-rewired/index.ts

```diff
@@ -353,6 +384,10 @@ const reactAppRewireConfig = {
     return (...args) => {
       const config = configFunction(...args)
       config.headers = headers
+      // enable hot module replacement! (HMR)
+      config.hot = true
```

Comment:
this shaves off a full browser reload on every compile. if it's buggy we can revert this line.